src/components/calcite-dropdown/calcite-dropdown.tsx

```diff
@@ -316,6 +318,8 @@ export class CalciteDropdown {
 
   mutationObserver = createObserver("mutation", () => this.updateItems());
 
+  resizeObserver = createObserver("resize", () => this.setMaxScrollerHeight());
```

Comment:
@anveshmekala we might just be able to call `reposition()` along with `this.setMaxScrollerHeight()` in order to fix https://github.com/Esri/calcite-components/issues/3234

Reply:
I verified that calling `this.reposition();` inside `setMaxScrollerHeight` fixes the issue.

@anveshmekala